src/features/settings/screens/UsersScreen/UsersScreen.tsx

```diff
@@ -51,23 +61,53 @@ const UsersScreen = () => {
     setShowEditModal(false);
   };
 
+  const handleAddUserModal = () => {
+    setShowAddModal(true);
+  };
+
   return (
     <>
-      <h1>{t("settingsUsers.title")}</h1>
-      <p>{t("settingsUsers.text")}</p>
+      <ScreenTitleView mainTitle={t("default.screenTitle")} />
+      <div className={styles.buttonContainer}>
+        <Button
+          type="primary"
+          size="large"
+          buttonText="Add user"
+          onClick={handleAddUserModal}
+        />
+      </div>
       <TableView
         dataSource={data}
         actionTitle={t("default.columnAction")}
         onEdit={handleEdit}
         onDelete={handleDelete}
+        onDuplicate={handleDuplicate}
       >
         <Table.Column
           key="name"
           dataIndex="name"
           title={t("settingsUsers.columnName")}
         />
       </TableView>
-      <UsersModal visible={showEditModal} onClose={handleCloseModal} />
+      <UsersModal
+        visible={showEditModal}
+        onClose={handleCloseModal}
+        onFormSubmit={handleCloseModal}
+      />
+
+      <FormModal
+        title="Add user modal"
+        visible={showAddModal}
+        onClose={() => setShowAddModal(false)}
+        onFormSubmit={values => {
+          console.log("Form submitted", values);
+          setShowAddModal(false);
+        }}
+      >
+        <Item name="Username" label="Username">
+          <Input type="text" />
+        </Item>
+      </FormModal>
```

Comment:
The `<FormModal` should be used inside the `UsersModal` and the `UsersModal` should then handle both edit and create mode